DSharpPlus.Test/TestBot.cs

```diff
@@ -42,19 +42,21 @@ public TestBot(TestBotConfig cfg, int shardid)
             {
                 AutoReconnect = true,
                 LargeThreshold = 250,
-                MinimumLogLevel = LogLevel.Debug,
+                MinimumLogLevel = LogLevel.Warning,
                 Token = this.Config.Token,
                 TokenType = TokenType.Bot,
                 ShardId = shardid,
                 ShardCount = this.Config.ShardCount,
                 MessageCacheSize = 2048,
-                LogTimestampFormat = "dd-MM-yyyy HH:mm:ss zzz"
+                LogTimestampFormat = "dd-MM-yyyy HH:mm:ss zzz",
+                Intents = DiscordIntents.All
```

Comment:
We can add intents to the test bot once we patch in v8.

Reply:
Ah crap, that didnt mean to go in, ill revert tonight